Tasks/CordovaCommand/lib/node-manager.js

```diff
@@ -48,11 +48,23 @@ function setupMaxNode(maxVersion, targetVersion, /*optional*/ installNpmOnWindow
         });
 }
 
+function useSystemNode() {
+    nodePath = path.dirname(taskLibrary.which('node'));
+    return Q();
+}
+
 function setupNode(targetVersion, /*optional*/ installNpmOnWindows) {
     if(!fs.existsSync(NODE_VERSION_CACHE)) {
         taskLibrary.mkdirP(NODE_VERSION_CACHE);
     }
-    var dlNodeCommand = new taskLibrary.ToolRunner(taskLibrary.which('curl', true));
+    
+    var curlPath = taskLibrary.which('curl', true);
+    if (!curlPath) {
+        taskLibrary.error("curl was not found in PATH. curl needs to be in the path on Windows. You can get curl by installing the Git Command Line Tools (www.git-scm.com/downloads)");
+        return null;
+    }
+    
+    var dlNodeCommand = new taskLibrary.ToolRunner(curlPath);
```

Comment:
Same comment as Cordova Build

There is one more fallback we need to consider unfortunately - that's if an environment variable is set. The variable varies by task name.

CORDOVA_DEFAULT_VERSION

So, order of precedence is:

1. Cordova version specified by task
2. Cordova version in taco.json
3. Env var
 4. else edge

It's the same logic that's in taco-team-build - we should consider rationalizing it when we pick up the other refactor task to reduce risk of breaking if vso-task-lib drops support for 0.12.7